tests/src/rules/no-unresolved.js

```diff
@@ -85,7 +92,7 @@ function runResolverTests(resolver) {
         options: [{ commonjs: true }] }),
     ],
 
-    invalid: [
+    invalid: [].concat(
```

Comment:
just curious why the `[].concat()` is needed. :)

Reply:
because RuleTester requires an array of objects, so there's no way to conditionally provide an ignoreable value, so we have to use `[].concat()` and have the conditional helper return an empty array, since concat flatMaps over arrays.